java/src/jmri/jmrix/AbstractMRNodeTrafficController.java

```diff
@@ -16,7 +16,7 @@
 public abstract class AbstractMRNodeTrafficController extends AbstractMRTrafficController {
 
     /**
-     * Creates a new instance of AbstractMRNodeTrafficController
+     * Ctor
```

Comment:
This Javadoc comment is worthless; providing a comment "this is a constructor" on a constructor does not document anything.

Reply:
A gooed example for documenting a TrafficController was lacking. Added starting with AbstractMRTRafficController